Makefile

```
@@ -93,6 +93,21 @@ doc: $(MANIFEST)
 	antsibull-docs collection --use-current --squash-hierarchy --dest-dir ./docs/plugins $(NAMESPACE).$(NAME)
 	make -C docs html
 
+branding:
+	sed -i 's/theforeman\.operations/redhat.satellite-operations/g' changelogs/config.yaml changelogs/changelog.yaml CHANGELOG.rst roles/*/README.md roles/*/*/*.yml
+	sed -i 's/foreman.example.com/satellite.example.com/g' roles/*/README.md roles/*/*/*.yml
+	sed -i 's#theforeman/foreman-operations-collection#RedHatSatellite/satellite-operations-collection#g' .github/workflows/*.yml
+	sed -i 's/theforeman-foreman/redhat-satellite-operations/g' .github/workflows/*.yml
+	sed -i 's/Foreman Operations Collection/Red Hat Satellite Operations Collection/g' docs/index.rst docs/conf.py
+	sed -i 's/The Foreman Project/Red Hat, Inc./g' docs/conf.py
+	sed -i '/FOREMAN_\w/ s/FOREMAN_/SATELLITE_/g' Makefile
+	sed -i '/foreman_proxy_\w/ s/foreman_proxy_/capsule_/g' roles/*/README.md roles/*/*/*.yml
+	sed -i '/foreman_\w/ s/foreman_/satellite_/g' roles/*/README.md roles/*/*/*.yml
+	sed -i 's/foreman-installer/satellite-installer/g' roles/*/README.md roles/*/*/*.yml
+	rm -rf roles/puppet_repositories roles/foreman_repositories roles/postgresql_upgrade
+	[ -d roles/foreman_proxy_certs_generate ] && mv roles/foreman_proxy_certs_generate roles/capsule_certs_generate || echo ''
```

```suggestion
	[ ! -d roles/foreman_proxy_certs_generate ] || mv roles/foreman_proxy_certs_generate roles/capsule_certs_generate
```

I always find it confusing to read `[ … ] && … || …` in one line, and the above at least removes one branch.

But that's totally a personal preference, so feel free to ignore.